logos/logos-workernode/logos_worker_node/calibration.py

```
@@ -1256,6 +1298,15 @@ def calibrate_model(
     # status so the master skips this worker until ops intervenes.
     _node_unhealthy_box: list[NodeTransientErrorPattern] = []
 
+    # Cap on per-probe ``--max-model-len`` shrink-and-retry attempts. vLLM's
+    # estimator is monotonically decreasing as the budget shrinks (it just
+    # divides KV bytes by per-token KV size), so a healthy run converges in
+    # 1–2 probes. Three gives slack for noisy floor-effects (graph capture
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)